enterprise/frontend/src/embedding-sdk/components/public/MetabotQuestion/MetabotChatEmbedding.module.css

```diff
@@ -16,7 +16,8 @@
 
 .container {
   position: relative;
-  width: 41.5rem;
+  max-width: 41.5rem;
```

Comment:
The text input wasn't responsive before, now it is 😄 
![image](https://github.com/user-attachments/assets/53cbb7ef-8b87-48a3-8e89-54273d66ffba)
![image](https://github.com/user-attachments/assets/4b18a641-8a9d-427b-a351-309e862fb142)